oak/client/application_client.h

```diff
@@ -59,13 +66,29 @@ class ApplicationClient {
   // Returns a grpc Channel connecting to the specified address, initialised with:
   // - Asylo channel credentials, possibly attesting the entire channel to the remote enclave
   // - Oak Policy call credentials, possibly attaching a specific Oak Policy to each call
+  //
+  // A fresh nonce is created for each newly created channel, and it is attached as an authorization
+  // bearer token to each outgoing call. Also a corresponding policy allowing access to the newly
+  // created token is attached to each outgoing call. The combined effect is that any interactions
```

Comment:
might help to add "is attached to each outgoing call (via the gRPC library invoking the attached PolicyMetadata object's GetMetadata method)."

Reply:
Done.